deploy/helm/sumologic/requirements.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: fluent-bit
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: 2.4.4
```

Comment:
how did you decide on these versions?

Reply:
This is now what I decided. This file is autogenerated by running `helm install`. It get the  version `2.4.4` from the version we specified for `version: ~2.4.0` in `requirements.yaml`.
https://stackoverflow.com/questions/53702445/use-of-requirements-lock-file-in-helm-charts
As here it shows that how it works:
1. After created initial `requirements.yaml` file. You run `helm install` and helm creates `requirements.lock` file as it builds the dependency tree.
2. On the next helm install, helm will ensure that it uses the same versions identified in the `requirements.lock` file.
3. At some later date, you update `requirements.yaml` file. You run `helm install` (or `helm upgrade`) and helm will notice your changes and update the `requirements.lock` file to reflect them.

Reply:
how did you decide to use `version: ~2.4.0` in the yaml file then? and is the auto-generated version `2.4.4` the latest version after `version: ~2.4.0` or something?

Reply:
`~2.4.0` is a patch-level version match (suggested by helm), which means  `>=2.4.0 and <2.5.0`. And `2.4.4` is auto generated then. Here I used the latest patch level version officially provided by fluent-bit. Maybe we need to ask @samjsong to make sure whether there is some explicit requirements for fluent-bit version.

Reply:
Don't know of any explicit requirements for fluent bit version, I've just been using latest for now, I'm assuming this is fine and can be upgraded in the future if there are updates we want to include